superset/assets/src/visualizations/nvd3/utils.js

```diff
@@ -96,6 +96,7 @@ export function generateRichLineTooltipContent(d, timeFormatter, valueFormatter)
   d.series.sort((a, b) => a.value >= b.value ? -1 : 1);
   d.series.forEach((series) => {
     const key = getFormattedKey(series.key, true);
+    let value = series.value != null ? valueFormatter(series.value) : "NA";
```

Comment:
do we also need an undefined check to restrict undefined values?

Reply:
Pandas converting missing points with null values. So we are taking care that points only.